### PR TITLE
Replace Webpack and Vite PNGs by SVGs

### PR DESCRIPTION
**Pull request by julien-deramond**
### Related PR

Linked to https://github.com/twbs/bootstrap/pull/36899

### Description

Replace Webpack and Vite PNGs by SVGs.

Note: I haven't found a SVG version for Parcel.

### Motivation & Context

The images weights is smaller:

| **Webpack**  |  **Vite**  |
|---|---|
| PNG: 1.9K | PNG: 4.0K | 
| SVG: 565B | SVG: 1.5K | 

#### [Live preview](https://deploy-preview-37199--twbs-bootstrap.netlify.app/)

Screenshot 2022-09-24 at 00 07 58

### CODE REVIEW

File: site/static/docs/5.2/assets/img/vite.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<svg width="410" height="404" viewBox="0 0 410 404" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ffoodd) [inline site/static/docs/5.2/assets/img/vite.svg:1]: I think `width`, `height` and `fill` are safe to drop, here. 🙂

Comment (julien-deramond) [reply site/static/docs/5.2/assets/img/vite.svg]: Good catch! I didn't even think to look at the content of the file :)

**Review by mdo — APPROVED** [review]
LGTM, but @XhmikosR probably has some image optimizations to apply ;).

**Comment by XhmikosR** [comment]
Thanks, I'll try to have a look at this in the next days!

**Pull request merged**